sift.js

```diff
@@ -214,7 +217,7 @@ var Sifter = (function() {
 
                 for(var i = 1; i < internal.networkAdapters.length; i++) {
                     var adapter = internal.networkAdapters[i];
-                    if (adapter.mac === plainAddress || (plainAddress && plainAddress.indexOf(adapter.mac) == 0)) {
+                    if (adapter.mac === plainAddress || (adapter.mac && plainAddress.indexOf(adapter.mac) == 0)) {
```

Comment:
`plainAddress` is never null now, but we don't want to match our new `null` address on the first index of every adapter mac